### PR TITLE
1.9.4 Release Notes WIP

### PR DESCRIPTION
**Pull request by sascala**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

File: src/releases.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
       "direct": "https://downloads.dcos.io/dcos/stable/1.9.3/dcos_generate_config.sh"
```
Comment (judithpatudith) [inline src/releases.json]: Lines 16-18 bump all these to 1.9.4

File: src/releases/index.jade [hunk 2]
```diff
@@ -16,14 +16,14 @@ block content
           .release-card.release-stable.col.col-12.md-col-6.md-pr1.relative
             div.bg-white.center.inner.px3.pt3.pb3.mb2.md-mb0
               h3.mb0 Latest Stable
-              p.release-version.pb1 DC/OS 1.9.3
+              p.release-version.pb1 DC/OS 1.9.4
               p.hero-list__item__copy__paragraph.release-copy Latest stable release. Tested and ready for use in production.
               a.cta.cta--text.release-notes.pb0(href='/releases/1.9.3') Release Notes
```
Comment (judithpatudith) [inline src/releases/index.jade]: bump to 1.9.4

**Comment by sascala** [comment]
retest this please

File: src/releases.json [hunk 3]
```diff
@@ -13,18 +13,19 @@
...
     "subreleases": [
+      { "name": "1.9.4", "url": "/releases/1.9.4/"},
       { "name": "1.9.3", "url": "/releases/1.9.3/"},
       { "name": "1.9.2", "url": "/releases/1.9.2/"},
       { "name": "1.9.1", "url": "/releases/1.9.1/"},
       { "name": "1.9.0", "url": "/releases/1.9.0/"},
       { "name": "1.9.0-rc4", "url": "/releases/1.9.0-rc4/"},
       { "name": "1.9.0-rc3", "url": "/releases/1.9.0-rc3/"}
-    ],
+    ],release-version.pb1
```
Comment (joerg84) [inline src/releases.json]: That usage looks off and seems to break the build.

Comment (joerg84) [reply src/releases.json]: IMO this shouldn't be in the json file....

Comment (judithpatudith) [reply src/releases.json]: Agreed

**Pull request merged**